tests/suites/1_stateful/04_mini_dataset/00_0001_mini_hits.sh

```diff
@@ -0,0 +1,59 @@
+#!/usr/bin/env bash
+
+CURDIR=$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)
+. "$CURDIR"/../../../shell_env.sh
+
+echo "drop table if exists hits;" | $MYSQL_CLIENT_CONNECT
+## Create table
+cat $CURDIR/../ddl/hits.sql | $MYSQL_CLIENT_CONNECT
+
+## Load data
+echo "COPY INTO ontime FROM 's3://repo.databend.rs/dataset/stateful/hits_1m.tsv.gz' credentials=(aws_key_id='$REPO_AWS_ACCESS_KEY_ID' aws_secret_key='$REPO_AWS_SECRET_ACCESS_KEY') FILE_FORMAT = ( type = 'CSV' field_delimiter = '\t' compression = 'gzip' record_delimiter = '\n' skip_header = 1 );" | $MYSQL_CLIENT_CONNECT
+
+## Run test
+echo 'SELECT COUNT(*) FROM hits;' | $MYSQL_CLIENT_CONNECT
```

Comment:
Write as array so that we don't need to write `echo xxx | xxx`.

For example:

https://github.com/datafuselabs/databend/blob/3e0ec788e14373b0925dac26475536159c272094/tests/suites/1_stateful/00_copy_v2/00_0000_copy_from_s3_location.sh#L11-L34